packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdInSummary.spec.ts

```diff
@@ -335,10 +438,30 @@ describeCompat("Create data store with group id", "NoCompat", (getTestObjectProv
 		assert(contextD.baseSnapshot !== undefined, "contextD should have a baseSnapshot");
 
 		assert.equal(callCount, 0, "Should not have made any network calls");
-		assertOmittedTree(contextA.baseSnapshot, loadingGroupId, "contextA tree not omitted");
-		assertOmittedTree(contextB.baseSnapshot, loadingGroupId, "contextB tree not omitted");
-		assertOmittedTree(contextC.baseSnapshot, loadingGroupId2, "contextC tree not omitted");
-		assertOmittedTree(contextD.baseSnapshot, loadingGroupId2, "contextD tree not omitted");
+		assertOmittedBlobContents(
+			contextA.baseSnapshot,
+			loadingGroupId,
+			blobContents,
+			"contextA tree not omitted",
+		);
+		assertOmittedBlobContents(
```

Comment:
If you just fixed assertOmittedTree, all these asserts would be 1 liners. Totally not required.

Reply:
Actually, both are testing different things. In assertOmittedTree(), the trees and blobs actually empty and that is when we fetched the grouped snapshot.